app/src/androidTest/java/com/mapbox/maps/testapp/annotation/UpdateAnnotationTest.kt

```diff
@@ -32,13 +32,16 @@ import java.util.concurrent.TimeoutException
 class UpdateAnnotationTest : BaseMapTest() {
   private val updateDelay = 100L
   private var index = 0
+  private var stopUpdate = false
```

Comment:
is this related to reset `nativeMap` to null ?

Reply:
Yes, we need to stop the runnable after activity is destroyed, otherwise will trigger `MapboxMapMemoryLeakException`

Reply:
I'm really worried that users might also face such cases in their apps. Do we really want to throw exceptions? Could we just print some error message and ignore the call instead of throwing?

Reply:
Keep a reference to mapboxMap and Style actually and invoke methods after destroy is wrong behavior and may cause MML issue.  We throw exceptions here to let users fix this issue.